core/server/master/src/main/java/alluxio/web/WebInterfaceGeneralServlet.java

```diff
@@ -107,6 +109,56 @@ public int getUsedSpacePercent() {
     }
   }
 
+  /**
+   * Class to make wrong property information more intuitive and human-readable.
+   */
+  public static final class WrongPropertyInfo {
+    /** The name of the property that has errors/warnings.*/
+    private String mName;
+    /**
+     * Record the values and corresponding hostnames.
+     * Each string in valuesAndHosts is of format Value (Host1, Host2, ...)
+     */
+    private List<String> mValuesAndHosts;
```

Comment:
I think the Web UI jsp files should not contain too complicated java codes. All the data transformation now is put in the web servlet files. It will be better for jsp files to do very easy get data and show data.

Reply:
Web UI directly deal with the ConfigReport in jsp. Thanks!